bin/alluxio-masters.sh

```diff
@@ -45,11 +45,16 @@ if [[ ${JOURNAL_TYPE} == "EMBEDDED" ]]; then
 fi
 for master in ${HOSTLIST[@]}; do
   echo "[${master}] Connecting as ${USER}..." >> ${ALLUXIO_TASK_LOG}
+  if [[ $master = "localhost" || $master = "127.0.0.1" ]]; then
+    ssh_command=""
+  else
+    ssh_command="ssh -o ConnectTimeout=5 -o StrictHostKeyChecking=no -tt ${master}"
+  fi
```

Comment:
Please define a function for this and share this function in `bin/alluxio-masters.sh` and `bin/alluxio-workers.sh` to avoid duplicated code

Something like this:

```
function ssh_command() {
    host=$1
    if [[ $host = "localhost" || $host = "127.0.0.1" ]]; then
     return ""
   else
     return "ssh -o ConnectTimeout=5 -o StrictHostKeyChecking=no -tt ${host}"
   fi
}
```

So this function can be called like this:

nohup $(ssh_command $master}  ${LAUNCHER} ...

Reply:
@huanghua78 thanks for your suggestion. I have updated the code. Please check once more when you are available.